### DISCUSSION TITLE
Can primsa + SQLite be used on Vercel?

### DISCUSSION THREAD

**Discussion by LGmatrix13** [prisma/prisma#6305] [Q&A] [answered]
Could I actually use SQLite with persistence on Vercel? As they use functions, I was I under her impression you could not...

**Comment by ryands17** [comment, marked as answer]
@LGmatrix13 👋 

I'm not sure about Vercel but yes it's possible to use SQLite with persistence on Lambda by keeping your file on a file system (like EFS) and writing to the file.

**Reply by LGmatrix13** [reply]
Thank you! I believe Vercel uses lambda so this is good to know.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
